pytest.ini

```diff
@@ -39,3 +39,4 @@ doctest_optionflags =
 asyncio_mode = auto
 markers=
     linkcheck: Test links
+    compatmode: Tests a feature of Cylc 7 compatibility mode
```

Comment:
I'm not sure how useful this marker is, especially as it's not applied to existing compatmode tests.